etl/steps/data/garden/antibiotics/2024-10-23/tracss.py

```diff
@@ -0,0 +1,139 @@
+        "B - System designed for surveillance of antimicrobial use, that includes monitoring national level sales† or use  of antibiotics in health services. ": "National monitoring of antimicrobial sales and antibiotic use in health services",
+        "C - Total sales of antimicrobials are monitored at national level and/or some monitoring of antibiotic use at sub-national level. ": "National monitoring of antimicrobial sales and some monitoring of antibiotic use at sub-national level",
+        "D - Prescribing practices and appropriate antibiotic use are monitored in a national sample of healthcare settings. ": "Prescribing practices and appropriate antibiotic use monitored in a sample of healthcare settings",
+        "E - On a regular basis (every year/two years) data is collected and reported on: a) and b) - refer to questionnaire for complete text": "Regular data collection and reporting",
+    }
+    check_keys_exist(tb, monitoring_consumption_human_3_2_responses, "monitoring_consumption_human_3_2")
+    tb["monitoring_consumption_human_3_2"] = tb["monitoring_consumption_human_3_2"].cat.rename_categories(
+        monitoring_consumption_human_3_2_responses
+    )
+
+    amr_surveillance_human_3_3_responses = {
+        "A - No capacity for generating data (antibiotic susceptibility testing and accompanying clinical and epidemiological data) and reporting on antibiotic resistance.  ": "No capacity for generating data on antibiotic resistance",
+        "B - AMR data is collated locally for common† bacterial infections in hospitalized and community patients‡, but data collection may not use a standardized approach and lacks national coordination and/or quality management. ": "Local collation of AMR data for common bacterial infections",
+        "C - AMR data are collated nationally for common bacterial infections in hospitalized and community patients, but national coordination and standardization are lacking. ": "National collation of AMR data for common bacterial infections",
+        "D - There is a standardized national AMR surveillance system collecting data on common bacterial infections in hospitalized and community patients, with established network of surveillance sites, designated national reference laboratory for AMR, and a national coordinating centre producing reports on AMR. ": "Standardized national AMR surveillance system",
+        "E - The national AMR surveillance system links AMR surveillance with antimicrobial consumption and/or use data for human health§. ": "National AMR surveillance system linked with antimicrobial consumption data",
+    }
+    check_keys_exist(tb, amr_surveillance_human_3_3_responses, "amr_surveillance_human_3_3")
+    tb["amr_surveillance_human_3_3"] = tb["amr_surveillance_human_3_3"].cat.rename_categories(
+        amr_surveillance_human_3_3_responses
+    )
+
+    woah_reporting_options_4_6_responses = {
```

Comment:
I see some of these are very long (though I see also they are not used for charts)